src/metabase/events/view_log.clj

```diff
@@ -57,6 +57,17 @@
       (catch Throwable e
         (log/warnf e "Failed to process view event. %s" topic)))))
 
+(derive ::dashboard-queried :metabase/event)
```

Comment:
not part of this PR but we're working on adding batch processing for `report_card.last_viewed_at, view_count` and alike in https://github.com/metabase/metabase/pull/45007

do you think we can batch processing this as well?

Reply:
Makes sense to me, thanks for implementing this. Would you want me to do that in this PR (waiting until after the other one is merged), or in a followup PR, or would you take that?

One slight concern is that if we stopped writing `last_used_at` or `last_viewed_at` entirely for a long period of time, we might recommend that people delete things they shouldn't. I know grouper is "best effort" without any guarantees. But I think it's probably still an acceptable tradeoff if the odds of dropping any particular job are low.